README.md

```diff
@@ -30,6 +30,27 @@ or using the `install` target in the `Makefile` by simply running
     make install
 
 
+## Computing p-values
+If you want perform a two sample test on your own samples X and Y you can use the function `pvalue()` in 
+`tests-wo-split/methods/pvalue`.
+A simple test of the validity of our method is to see wheter the p-values are uniformly distributed 
+under the null hypothesis (samples come from the same distritbution).
+#####Example: uniform distribution of p-values
```

Comment:
This doesn't work that well with github's `markdown` renderer :) If you remove this line at all, the rest would be parsed as a code block and would be rendered correctly.

Reply:
Sorry, I am not sure if I understand. Which line do you mean? 35? Could you implement it the way it should be? I was already satisfied with the way it was rendered :D

Reply:
Check here and you will see that it is not rendered properly:
https://github.com/MPI-IS/tests-wo-splitting/tree/jonas_pvalue_readmen

Reply:
You need to remove a hash and add a space:
`#### Example:...`

Reply:
Ok, done.